tests/unit/lms/services/h_api_test.py

```diff
@@ -61,17 +57,15 @@ def test_create_user_works(self, h_api, h_user):
             },
         )
 
-    def test_update_user_works(self, h_api, h_user):
+    def test_update_user_works(self, h_api, h_user, _api_request):
         h_api.update_user(h_user)
 
-        h_api._api_request.assert_called_once_with(
+        _api_request.assert_called_once_with(
             "PATCH",
             "users/sentinel.username",
             data={"display_name": h_user.display_name},
         )
 
-    @patch.object(HAPI, "create_user")
-    @patch.object(HAPI, "update_user")
```

Comment:
Replaced these with pytest fixtures to be more consistent with how we patch things throughout our tests (including other patch fixtures in this same test module) and because pytest fixtures are more powerful as they can compose with other fixtures